src/MPQEditor/MPQData.ts

```diff
@@ -139,6 +140,14 @@ export class MPQData {
     return layersToAdd;
   }
 
+  getVisqPath(): string {
```

Comment:
FYI, typescript provides 'get' 'set' keywords for easier getter/setter implementation.

Reply:
> (optional) FYI, typescript provides 'get' 'set' keywords for easier getter/setter implementation. 😄

@dayo09 
Thank you! I've changed the draft and this PR.